examples/3d-tiles/tile-3d-layer.js

```diff
@@ -289,13 +306,43 @@ export default class Tile3DLayer extends CompositeLayer {
     // tileHeader.userData = {gltfObjects};
   }
 
+  _calculatePositionsOffset(tileHeader) {
+    const {center} = tileHeader.boundingVolume;
+    const {distanceScales} = this.context.viewport;
+
+    const {metersPerPixel, pixelsPerDegree} = distanceScales;
+
+    const {pointsCount} = tileHeader.userData;
+    const {positions} = tileHeader.content;
+
+    for (let i = 0; i < pointsCount; i++) {
+      const point = new Vector3(positions[i * 3], positions[i * 3 + 1], positions[i * 3 + 2]);
```

Comment:
Creating a new `Vector3` every iteration will hurt performance. Move outside of loop and use `point.set(x,y,z)` here.